examples/web-instrumentation/README.md

```diff
@@ -0,0 +1,384 @@
+# Instrumenting Web Applications End-to-End with Stackdriver and OpenTelemetry
+
+The app is something like a simple web version of Apache Bench. It includes
+JavaScript browser code that drives HTTP requests to a Node.js backend that
+can be run anywhere that you can run Node.
+
+Create a Google Cloud Project with billing enabled. Install Node.js.
+Install the Google Cloud SDK.
+
+## Setup
+
+Clone this repository to your environment with the command
+
+```shell
+git clone https://github.com/GoogleCloudPlatform/professional-services.git
+```
+
+Change to this directory and set an environment variable to remember the
+location
+
+```shell
+cd professional-services/examples/web-instrumentation
+WI_HOME=`pwd`
+```
+
+Set Google Cloud SDK to the current project 
```

Comment:
Should we link to how to docs on how to set up a project if they don't have one?

Reply:
Yes. I have added that in the enumerated list mentioned above